docs/robot/virtual-robot/robot-class.md

```diff
@@ -20,11 +20,166 @@ gh_link: docs/robot/virtual-robot/robot-class.md
 ---
 
 ## Introduction
+The Robot Class implements a collection of sensors, communication methods, output devices, Helper functions to emulate a real hardware robot in the virtual space.
+
+### Robot(int id, double x, double y, double heading, char reality)
+<br>
+
+	id      - id of the robot (unique in virtual and real)
+	x       - x cordinates on the arena
+	y       - y cordinates on the arena
+	heading - 
```

Comment:
heading-The absolute angle of the robot from the positive direction of the x axis